end-to-end-tests/cypress/support/commands.js

```diff
@@ -23,4 +23,54 @@
 //
 // -- This will overwrite an existing command --
 // Cypress.Commands.overwrite('visit', (originalFn, url, options) => { ... })
-import '@testing-library/cypress/add-commands';
\ No newline at end of file
+import '@testing-library/cypress/add-commands';
+
+Cypress.Commands.add('navToViewer', (viewerLinkText, titleText) => {
+  cy.visit(Cypress.env('base_url'))
+  cy.findByText('OSCAL Catalog Viewer').should('exist')
+  cy.get('button').first().click()
+  cy.contains(viewerLinkText).click()
+  cy.findByText(titleText).should('be.visible')
+})
+
+Cypress.Commands.add('navToSspViewer', () => {
+  cy.navToViewer(
+    'System Security Plan Viewer',
+    'OSCAL System Security Plan Viewer'
+  ) 
+})
+
+Cypress.Commands.add('navToTestSspRestMode', (sspTitle) => {
+  cy.navToSspViewer()
+  cy.contains('Select OSCAL SSP').parent().click()
+  cy.contains(sspTitle).click()
+  cy.contains(sspTitle).should('be.visible')
+})
+
+Cypress.Commands.add('navToCdefViewer', () => {
+  cy.navToViewer(
+    'Component Viewer',
+    'OSCAL Component Viewer'
+  ) 
+})
+
+Cypress.Commands.add('getInputByLabel', (label) => {
+  cy.contains('label', label)
+    .invoke('attr', 'for')
+    .then((id) => {
+      //cy.get('#' + id)
+      cy.get(`input[id="${id}"]`)
+    })
+})
+
+Cypress.Commands.add('getTestSspJson', () => {
+  cy.request('GET', `${Cypress.env('api_url')}/oscal/v1/system-security-plans/cff8385f-108e-40a5-8f7a-82f3dc0eaba8`).then(
```

Comment:
`/oscal/v1` should probably be put into the `api_url` config. Or `api_url` should be renamed to `api_origin`. But we probably want `api_url` to be about equivalent to `REACT_APP_API_BASE_URL` for `oscal-react-library`.

Reply:
Yep, good call.  Updated.